operational/vmware/instance_tag_sync/instance_tag_sync.pt

```diff
@@ -2,9 +2,15 @@ name "VMWare Instance Tag Sync"
 rs_pt_ver 20180301
 type "policy"
 short_description "Adds tags to vmware instances from CMP. See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/vmware/instance_tag_sync) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.3"
+long_description ""
 severity "medium"
 category "Operational"
+info(
+  version: "1.4",
+  provider: "VMWare",
```

Comment:
The provider should be "Flexera Cloud Management" since the policy is calling RightScale Cloud management API's

Reply:
made the change